test/Bridges/Objective/slack.jl

```diff
@@ -201,6 +203,244 @@ function test_SlackBridge_ObjectiveFunctionValue()
+    cquad = MOI.get(
+        mock,
+        MOI.ListOfConstraintIndices{
+            MOI.ScalarQuadraticFunction{Float64},
+            MOI.GreaterThan{Float64},
+        }(),
+    )
+    MOI.set(bridged_mock, MOI.ConstraintName(), cquad[1], "quad")
+
+    s = """
+    variables: x, y, s
+    x >= 1.0
+    y >= 2.0
+    quad: 1.0 * x * x + 1.0 * x * y + 1.0 * y * y + -1.0 * s >= 0.0
+    maxobjective: s
+    """
+    model = MOI.Utilities.Model{Float64}()
+    MOI.Utilities.loadfromstring!(model, s)
+    MOI.Utilities.test_models_equal(
+        mock,
+        model,
+        [var_names; "s"],
+        ["quad"],
+        [
+            ("x", MOI.GreaterThan{Float64}(1.0)),
+            ("y", MOI.GreaterThan{Float64}(2.0)),
+        ],
+    )
+    s = """
+    variables: x, y
+    x >= 1.0
+    y >= 2.0
+    maxobjective: 1.0 * x * x + 1.0 * x * y + 1.0 * y * y
+    """
+    model = MOI.Utilities.Model{Float64}()
+    MOI.Utilities.loadfromstring!(model, s)
+    MOI.Utilities.test_models_equal(
+        bridged_mock,
+        model,
+        var_names,
+        String[],
+        [
+            ("x", MOI.GreaterThan{Float64}(1.0)),
+            ("y", MOI.GreaterThan{Float64}(2.0)),
+        ],
+    )
+    _test_delete_objective(
+        bridged_mock,
+        2,
+        (
+            (MOI.ScalarQuadraticFunction{Float64}, MOI.GreaterThan{Float64}, 0),
+            (MOI.ScalarQuadraticFunction{Float64}, MOI.LessThan{Float64}, 0),
+        ),
+    )
+    return
+end
+
+function test_quadratic_integration()
+    mock = MOI.Utilities.MockOptimizer(MOI.Utilities.Model{Float64}())
+    bridged_mock = MOI.Bridges.Objective.Slack{Float64}(mock)
+    MOI.Utilities.set_mock_optimize!(
+        mock,
+        (mock::MOI.Utilities.MockOptimizer) -> MOI.Utilities.mock_optimize!(
+            mock,
+            [4 / 7, 3 / 7, 6 / 7, 13 / 7],
+            (MOI.ScalarAffineFunction{Float64}, MOI.GreaterThan{Float64}) =>
+                [5 / 7, 6 / 7],
+        ),
+    )
+    MOI.Test.test_quadratic_integration(bridged_mock, MOI.Test.Config())
+    return
+end
+
+function test_quadratic_duplicate_terms()
+    mock = MOI.Utilities.MockOptimizer(MOI.Utilities.Model{Float64}())
+    bridged_mock = MOI.Bridges.Objective.Slack{Float64}(mock)
+    MOI.Utilities.set_mock_optimize!(
+        mock,
+        (mock::MOI.Utilities.MockOptimizer) -> MOI.Utilities.mock_optimize!(
+            mock,
+            [4 / 7, 3 / 7, 6 / 7, 13 / 7],
+            (MOI.ScalarAffineFunction{Float64}, MOI.GreaterThan{Float64}) =>
+                [5 / 7, 6 / 7],
+        ),
+        (mock::MOI.Utilities.MockOptimizer) -> MOI.Utilities.mock_optimize!(
+            mock,
+            [4 / 7, 3 / 7, 6 / 7, -2 * 13 / 7],
+            (MOI.ScalarAffineFunction{Float64}, MOI.GreaterThan{Float64}) =>
+                [10 / 7, 12 / 7],
+        ),
+    )
+    MOI.Test.test_quadratic_duplicate_terms(bridged_mock, MOI.Test.Config())
+    return
+end
+
+function test_()
```

Comment:
Is the name of this function intentional ?

Reply:
Oops.